org.baedert.liblistbox.json

```diff
@@ -0,0 +1,80 @@
+{
+    "app-id" : "org.baedert.liblistbox",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "demo",
+    "finish-args" : [
+        "--socket=wayland",
+        "--share=network"
+    ],
+    "build-options" : {
+        "cflags" : "-g",
+        "env" : {
+            "GTK_INTERACTIVE" : "true"
+        }
+    },
+    "modules" : [
+        {
+            "name" : "wayland",
+            "buildsystem" : "autotools",
+            "builddir" : true,
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/wayland-project/wayland.git",
+                    "tag" : "1.15.0"
+                }
+            ],
+            "config-opts" : [
+                "--disable-documentation"
+            ]
+        },
+        {
+            "name" : "graphene",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/ebassi/graphene.git",
+                    "tag" : "1.8.2"
+                }
+            ],
+            "config-opts" : [
+                "--libdir=lib",
+                "-Dintrospection=false",
+                "-Dtests=false",
+                "-Dbenchmarks=false"
+            ]
+        },
+        {
+            "name" : "gtk-4.0",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "config-opts" : [
+                "--prefix=/app",
+                "--libdir=lib",
+                "-Dintrospection=false",
+                "-Ddemos=false",
+                "-Dbuild-tests=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://download.gnome.org/sources/gtk+/3.94/gtk+-3.94.0.tar.xz",
```

Comment:
If people use this to build it themselves, it should probably just use gtk+ master since I'll be adapting liblistbox to that and not keep it at the state of 3.94.